### PR TITLE
Fix for period following Latin text in Arabic string

### PR DESCRIPTION
**Pull request by matthew-ivre**
https://vreducation.atlassian.net/browse/EN-43419

Corrects issue highlighted in orange:
image

Result:
image

### CODE REVIEW

**Comment by matthew-ivre** [comment]
Continuing to update both the project and UPMPackage folders until we decide on how to restructure.

**Comment by matthew-ivre** [comment]
All tests passed.

**Review by daniel-ivre — APPROVED** [review]

**Review by juanma-ivre — APPROVED** [review]

File: UPMPackage/Tests/EngageOrderingTests.cs
Comment (ruben-ivre) [inline UPMPackage/Tests/EngageOrderingTests.cs:1]: We have a warning that this script is missing is .meta companion, it's not a big deal, but would be nice if we can fix it.

**Review by ruben-ivre — CHANGES_REQUESTED** [review]

File: UPMPackage/Tests/EngageOrderingTests.cs
Comment (matthew-ivre) [reply UPMPackage/Tests/EngageOrderingTests.cs:1]: Sorry, I missed this comment. I've added the .meta files.

**Review by ruben-ivre — APPROVED** [review]

**Pull request merged**